aspnetcore/blazor/index.md

```diff
@@ -57,102 +57,21 @@ Using .NET for client-side web development offers the following advantages:
+<!-- Start >=8.0 coverage -->
 
 :::moniker range=">= aspnetcore-8.0"
 
-```razor
-@page "/"
-@attribute [RenderModeServer]
-
-<h1>Hello, world!</h1>
-
-<p>
-    Welcome to your new app.
-</p>
-
-<Dialog Title="Learn More">
-    Do you want to <i>learn more</i> about Blazor?
-</Dialog>
-```
-
-:::moniker-end
-
-:::moniker range="< aspnetcore-8.0"
-
-```razor
-@page "/"
-
-<h1>Hello, world!</h1>
-
-<p>
-    Welcome to your new app.
-</p>
-
-<Dialog Title="Learn More">
-    Do you want to <i>learn more</i> about Blazor?
-</Dialog>
-```
-
-:::moniker-end
-
-The dialog is rendered when the `Index` component is accessed in a browser. When the button is selected by the user, the browser's developer tools console shows the message written by the `OnYes` method:
-
-![Dialog component rendered in the browser nested inside of the Index component. The browser developer tools console shows the message written by C# code when the user selects the Yes! button in the UI.](~/blazor/index/_static/dialog.png)
-
-Components render into an in-memory representation of the browser's [Document Object Model (DOM)](https://developer.mozilla.org/docs/Web/API/Document_Object_Model/Introduction) called a *render tree*, which is used to update the UI in a flexible and efficient way.
-
-:::moniker range=">= aspnetcore-8.0"
-
-## Blazor Web Apps
+## Build a full-stack web app with Blazor
 
 Blazor Web Apps provide a component-based architecture with server-side rendering and full client-side interactivity in a single project, where you can switch between server-side and client-side rendering modes and even mix them in the same page. 
 
 When a Blazor Web App uses interactivity with server rendering, the server generates HTML in response to a request and sends it to the browser. The page loads fast because UI rendering is performed quickly on the server without the need to download a large JavaScript bundle or wait for the establishment of a [SignalR](xref:signalr/introduction) connection to the client.
 
 *Streaming rendering* can improve the user experience with SSR when long-running asynchronous tasks are required to fully render a page. Initially, Blazor renders the entire page for the browser with placeholder content. The asynchronous operations execute on the server. After the operations are complete, the updated content is sent to the browser on the same response connection and patched into page. The benefit of this approach is that the main layout of the app renders as quickly as possible.
```

Comment:
Diving deep into streaming rendering here feels like a bit too much detail. See my suggestion above for a way to mention streaming rendering as a progressive enhancement to static server rendering.